<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XVideoUIClass</class>
 <widget class="QWidget" name="XVideoUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XVideoUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(120, 120, 120, 255), stop:0.495 rgba(39, 39, 39, 255), stop:0.505 rgba(39,39, 39, 255), stop:1 rgba(120, 120,120, 255));
 border: none;
border-radius:5px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }
QPushButton:!hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(39, 39, 39, 255), stop:0.495 rgba(120, 120, 120, 255), stop:0.505 rgba(120,120, 120, 255), stop:1 rgba(39, 39,39, 255));
 border: none;
border-radius:5px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }
#XVideoUIClass
{
	background-color: rgb(53, 53, 53);
}
#closeButton
{
font: 75 11pt &quot;等线&quot;;
color: rgb(255, 255, 255);
background-color: rgb(53, 53, 53);
}


QLabel
{
	
	color: rgb(255, 255, 255);
	
	font: 75 12pt &quot;黑体&quot;;
}
</string>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="XVideoWidget" name="src1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>380</width>
     <height>280</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>340</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QSlider" name="playSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>亮度[0-100]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>对比度[1.0-3.0]</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="bright">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>390</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="contrast">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>420</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>3.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="setButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>390</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="XVideoWidget" name="des">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>380</width>
     <height>280</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XVideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>xvideowidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="XVideoUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>XVideoUIClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>782</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>XVideoUIClass</receiver>
   <slot>Open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>XVideoUIClass</receiver>
   <slot>SliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>XVideoUIClass</receiver>
   <slot>SliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>XVideoUIClass</receiver>
   <slot>SetPos(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setButton</sender>
   <signal>clicked()</signal>
   <receiver>XVideoUIClass</receiver>
   <slot>Set()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>714</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>-31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportButton</sender>
   <signal>clicked()</signal>
   <receiver>XVideoUIClass</receiver>
   <slot>Export()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>571</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Open()</slot>
  <slot>SliderPress()</slot>
  <slot>SliderRelease()</slot>
  <slot>SetPos(int)</slot>
  <slot>Set()</slot>
  <slot>Export()</slot>
 </slots>
</ui>
